data/crac/crac-io/crac-io-cim/src/test/java/com/powsybl/openrao/data/crac/io/cim/craccreator/CimCracCreatorTest.java

```diff
@@ -527,6 +527,7 @@ void testImportNetworkActions() throws IOException {
         assertRemedialActionNotImported("PRA_23", ImportStatus.INCONSISTENCY_IN_DATA);
         assertRemedialActionNotImported("PRA_24", ImportStatus.ELEMENT_NOT_FOUND_IN_NETWORK);
         assertRemedialActionNotImported("PRA_25", ImportStatus.NOT_YET_HANDLED_BY_OPEN_RAO);
+        assertRemedialActionNotImported("ARA_28", ImportStatus.INCONSISTENCY_IN_DATA);
```

Comment:
I created an ARA instead of a PRA because I need to associate it to a contingency.
PS: I can change its number to 26, and also rename PRA_26 and PRA_27 to 27 and 28 if needed.